### ISSUE TITLE
Fresh Install Fails

### ISSUE THREAD

**Issue by rc1891995** [sans-dfir/sift#154]
taking a class and instructor is using SIFT3.  I run Ubuntu 16 natively, but didn't want to use my primary OS.  So I loaded fresh .iso of Ubuntu 16.04LTS into Oracle Virtual Box 5.1.22.  Ubuntu loaded fine.  ran update. no issue.
ran the "quiet" version of wget to change to a SIFT workstation.  nothing happened.  removed the quiet.  below is what I got...ERROR 404: Not Found

got the command from: https://digital-forensics.sans.org/community/downloads

Please advise...thank you.

david@david-VirtualBox:~$ wget --quiet -O - https://raw.github.com/sans-dfir/sift-bootstrap/master/bootstrap.sh | sudo bash -s -- -i -s -y
[sudo] password for david: 
david@david-VirtualBox:~$ wget -O - https://raw.github.com/sans-dfir/sift-bootstrap/master/bootstrap.sh | sudo bash -s -- -i -s -y
--2017-06-17 08:23:16--  https://raw.github.com/sans-dfir/sift-bootstrap/master/bootstrap.sh
Resolving raw.github.com (raw.github.com)... 151.101.112.133
Connecting to raw.github.com (raw.github.com)|151.101.112.133|:443... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: https://raw.githubusercontent.com/sans-dfir/sift-bootstrap/master/bootstrap.sh [following]
--2017-06-17 08:23:17--  https://raw.githubusercontent.com/sans-dfir/sift-bootstrap/master/bootstrap.sh
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 151.101.112.133
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|151.101.112.133|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2017-06-17 08:23:18 ERROR 404: Not Found.

**Comment by ekristen** [comment]
Thanks for the write up, I apologize for the confusion. We are in the process or releasing a new version of SIFT built on 16.04 and before we got all the new instructions updated I removed the bootstrap script.  I've added it back with instructions on how to continue to build on 14.04 or how to build the new 16.04 SIFT, so now if anyone runs the command they will get output in their terminal with more information.
